docs/integrations/sources/chargebee.md

```diff
@@ -31,7 +34,20 @@ Some streams may depend on Product Catalog version and be accessible only on sit
     - Plans
     - Addons
     
-3. presented only in `Product Catalog 2.0` will be added soon.
+3. presented only in `Product Catalog 2.0`:
+    - Items
+    - Item Prices
+    - Attached Items
+
+Also, 8 streams from the above 9 incremental streams are pure incremental meaning that they:
+- read only new records;
+- output only new records.
+
+`Attached Items` incremental stream is also incremental but with one difference, it:
+- read all records;
+- output only new records.
+
+Please, consider this behaviour when using `Attached Items` incremental stream because it may affect you API call limits.
```

Comment:
This is the most important sentence here, because we want a non-engineer to easily understand this. Made a suggestion to make it super clear
```suggestion
This means that syncing the `Attached Items` stream, even in incremental mode, is expensive in terms of your Chargebee API quota. Generally speaking, it incurs a number of API calls equal to the total number of attached items in your chargebee instance divided by 100, regardless of how many AttachedItems were actually changed or synced in a particular sync job. 
```